src/table/table.scss

```diff
@@ -122,265 +236,315 @@
+      );
     }
-    user-select: none;
 
-    &:hover {
+    &::selection {
       @include themed {
-        background-color: t(iui-color-background-3) !important;
+        background-color: rgba(
+          t(iui-color-foreground-positive-rgb),
+          t(iui-opacity-4)
+        );
       }
     }
+  }
+    
+  &.iui-warning {
+    @include themed {
+      background-color: rgba(
+        t(iui-color-foreground-warning-rgb),
+        t(iui-opacity-6)
+      );
 
-    > .iui-content {
-      @include themed {
-        border-bottom: solid 1px t(iui-color-background-4);
-      }
-      width: 100%;
-      display: flex;
-      justify-content: space-around;
-      align-items: center;
-      padding: $iui-table-density;
-      flex: 1;
-
-      > .iui-full-line {
-        flex: 1;
-      }
-
-      > .iui-controls {
+      &::selection {
         @include themed {
-          color: t(iui-text-color);
-        }
-        width: 300px;
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        flex: 1;
-
-        > a {
-          @include iui-anchor;
-          margin: 0 $iui-s;
-        }
-
-        > .iui-ellipsis {
-          margin: 0 $iui-s;
-        }
-
-        > .iui-current,
-        > .iui-ellipsis {
-          @include themed {
-            color: t(iui-text-color);
-          }
-          cursor: default;
+          background-color: rgba(
+            t(iui-color-foreground-warning-rgb),
+            t(iui-opacity-4)
+          );
         }
+      }
+    }
+  }
 
-        > .iui-current {
-          text-decoration: underline;
-          font-weight: $iui-font-weight-semibold;
-        }
+  &.iui-negative {
+    @include themed {
+      background-color: rgba(
+        t(iui-color-foreground-negative-rgb),
+        t(iui-opacity-6)
+      );
+    }
 
-        > a > .iui-icon {
-          @include themed {
-            color: t(iui-icons-color-actionable);
-            fill: t(iui-icons-color-actionable);
-          }
-          display: block;
-          width: $iui-icons-default;
-          height: $iui-icons-default;
+    &::selection {
+      @include themed {
+        background-color: rgba(
+          t(iui-color-foreground-negative-rgb),
+          t(iui-opacity-4)
+        );
+      }
+    }
+  }
 
-          cursor: pointer;
+  &.iui-editable {
+    cursor: pointer;
+    @include iui-focus;
 
-          &:hover {
-            @include themed {
-              color: t(iui-icons-color-actionable-hover);
-              fill: t(iui-icons-color-actionable-hover);
-            }
-          }
-        }
+    &:hover {
+      @include themed {
+        box-shadow: inset 0 0 0 1px t(iui-color-foreground-primary);
       }
+    }
+  }
+}
 
-      > .iui-location {
-        display: flex;
-        justify-content: flex-end;
-        align-items: center;
-        flex: 1;
-
-        > .iui-input-container {
-          margin: 0;
+@mixin iui-table-row-status($status: positive) {
```

Comment:
not having a default status and passing explicitly would be more preferred way as it is not obvious, what is the default value (for example, I would not agree it should be positive.. informational - maybe).
From code readiness its better too to see what status you are passing explicitly to that mixin to get the styles.

Reply:
Vilius left a similar content on a previous PR.  I'm still pretty new to mixins and am learning the best practices of using it.  I'll remove the default positive status and require it to be declared when included.